src/github/api/mod.rs

```diff
@@ -116,18 +119,22 @@ pub struct RepositoryClient {
     /// The client caches the access token for this given repository and refreshes it once it
     /// expires.
     client: Octocrab,
-    repository: GithubRepoName,
+    // We store the name separately, because repository has an optional owner, but at this point
+    // we must always have some owner of the repo.
+    repo_name: GithubRepoName,
+    repository: Repository,
 }
 
 impl RepositoryClient {
-    pub async fn post_pr_comment(&self, issue: u64, content: &str) -> octocrab::Result<Comment> {
```

Comment:
I think it would be nice to still have this method that calls to the new free function.

Reply:
I'd like to keep the client be a dumb holder of token/connection/repository ID. If there will be a lot of git operations we can put them behind into struct impl later.